packages/embed/src/utils/create-iframe/create-iframe.ts

```diff
@@ -78,4 +78,4 @@ export const createIframe = (formId: string, type: EmbedType, options: CreateIfr
   return { iframe, embedId, refresh, focus }
 }
 
-type CreateIframeOptions = UrlOptions & ActionableOptions & IframeOptions
+export type CreateIframeOptions = UrlOptions & ActionableOptions & IframeOptions
```

Comment:
This export was added here but the type itself is not being used anywhere else in the PR